modules/swagger-codegen/src/main/java/io/swagger/codegen/utils/ModelUtils.java

```diff
@@ -0,0 +1,29 @@
+package io.swagger.codegen.utils;
+
+import io.swagger.codegen.CodegenModel;
+
+import java.util.List;
+import java.util.Map;
+
+public class ModelUtils {
+    public static CodegenModel getModelByName(final String name, final Map<String, Object> models) {
```

Comment:
@harishchoragudi It would be nice to add a proper docstring explaining what this function does.